databuilder/models/table_metadata.py

```diff
@@ -157,24 +167,23 @@ def __init__(self,
                  description: Union[str, None],
                  col_type: str,
                  sort_order: int,
-                 badges: Union[List[str], None] = None
+                 badges: Union[List, str, None] = None,
```

Comment:
badges should never be just a string, it will always be a list or nothing

Reply:
Right, I now changed the call signature to put the responsibility to form the list on the caller instead of allowing a string as well.

My original thought with adding string as an option was to apply the [robustness principle](https://en.wikipedia.org/wiki/Robustness_principle): "Be conservative in what you send, _be liberal in what you accept_"

Thank you for reviewing @allisonsuarez